src/handlers/Registrar.ts

```diff
@@ -114,12 +119,16 @@ export const makeRegistryHandlers = (baseName: `.${`${string}`}eth`) => {
       args: { tokenId, from, to },
     }: {
       context: Context;
-      args: Event<NsType<"BaseRegistrar:Transfer">>["args"];
+      args: {
```

Comment:
and i think for each of these functions in `Registrar.ts` we should standardize the api to be like a ponder handler: it always accepts context and event, and `args` is within the event. we need not do this in this pr, i can manage it once we've merged if you'd like